src/dataset-builder/CohortEditor.ts

```diff
@@ -37,15 +40,30 @@ const narrowMargin = 5;
 const wideMargin = 10;
 
 type CriteriaViewProps = {
-  criteria: AnyCriteria;
-  deleteCriteria: (criteria: AnyCriteria) => void;
-  updateCriteria: (criteria: AnyCriteria) => void;
+  readonly datasetId: string;
+  readonly criteria: AnyCriteria;
+  readonly deleteCriteria: (criteria: AnyCriteria) => void;
+  readonly updateCriteria: (criteria: AnyCriteria) => void;
 };
 
 const addCriteriaText = 'Add criteria';
 
-export const CriteriaView = ({ criteria, deleteCriteria, updateCriteria }: CriteriaViewProps) =>
-  div(
+export const CriteriaView = (props: CriteriaViewProps) => {
+  const { datasetId, criteria, deleteCriteria, updateCriteria } = props;
+
+  const [criteriaCount, setCriteriaCount] = useLoadedData<DatasetParticipantCountResponse>();
+
+  useEffect(() => {
+    setCriteriaCount(async () =>
+      DataRepo()
+        .dataset(datasetId)
+        .getCounts({
+          cohorts: [{ criteriaGroups: [{ criteria: [criteria], name: '', meetAll: true, mustMeet: true }], name: '' }],
```

Comment:
Do the names not matter?

Reply:
Yes, the API doesn't need to accept `name` for the cohort or criteria group since they're ignored by `getCounts()`.

If at some point we add server-side saving of the "wizard" state then we'd need to send them, but it'd be clearer to remove them as part of the API cleanup fo rnow.

Reply:
Why are meetAll and mustMeet always true instead of being derived from the user selection on the UI?

Reply:
The value computed here is independent of any other UI selection; the UI needs to show the roll-up count of this criteria regardless of what other criteria are selected, or what its criteria group settings are.

Since the `getCounts()` API only accepts `Cohort[]` we need to make an object of that type to satisfy the API. So this code creates a cohort on-the-fly with stub values. The value of `meetAll` doesn't matter since there's only one entry. The value of `mustMeet` must be `true` or else the value shown would be the inverse of the selection. I can add a comment here.